app.py

```
@@ -159,23 +180,300 @@ def render_settings(action):
+
+        # Update skip_variables in global section
+        if 'global' not in config_data:
+            config_data['global'] = {}
+        config_data['global']['config_diff_skip_variable'] = skip_variables
+
+        # Save config back to file
+        _atomic_write(config, yaml.dump(config_data, default_flow_style=False, sort_keys=False))
+
+        logging.info(f"Updated config_diff_skip_variable: {skip_variables}")
+        return jsonify({'success': True})
+    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)